packages/fast-development-site-react/src/components/site/component-wrapper.tsx

```diff
@@ -34,12 +34,9 @@ const styles: ComponentStyles<IComponentWrapperManagedClasses, IDevSiteDesignSys
         borderBottom: componentWrapperBorder,
     },
     componentWrapperExamples: {
-        "&:last-child:not(:nth-child(3n + 3)), &:nth-child(3n + 1)": {
+        "&:nth-child(3n + 1), &:nth-child(3n + 2)": {
```

Comment:
Going to pull this down for visual confirmation to confirm localization doesn't break this.

Reply:
@eljefe223 I think this only fixes part of the problem. This ends up breaking in RTL, so we likely need to support localization here as well.

Reply:
That is because this fix is in coordination with the other fix. You need to pull in my other RTL changes.